app/views/tag/_tags_popover.html.erb

```diff
@@ -3,10 +3,10 @@
   <%= form_with remote: true, url: '/subscribe/multiple/tag' do |f| %>
     <% tags.each do |tag| %>
     <div>
-      <input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>" <% if current_user.following(tag.name) %> checked <% end %> ><span class="<% if tag.name.include?(':') %> label label-primary <% else %> label label-default <% end %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %></span>
+      <input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>" <% if current_user && current_user.following(tag.name) %> checked <% end %> ><span class="<% if tag.name.include?(':') %> label label-primary <% else %> label label-default <% end %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %></span>
     </div>
     <% end %>
-      <%= f.submit 'Subscribe', class: "btn btn-primary btn-sm", style: 'margin-top: 10px;' %>
+      <%= f.submit 'Subscribe', class: "requireLogin btn btn-primary btn-sm", style: 'margin-top: 10px;' %>
```

Comment:
OK, so the above code or something like it should now work, because `require_login_and_redirect_to()` now exists because of #4739. @IshaGupta18 did you want to try finishing this one up?

Reply:
Sure @jywarren I think I kinda lost track of this one, I am so sorry. I'll get caught up with it and finish it off! Thanks a lot!